daemon/manager/logs/logs.go

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2024 The Kmesh Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at:
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package logs
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"kmesh.net/kmesh/pkg/logger"
+)
+
+func NewCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "logs",
+		Short:   "Print kmesh-daemon's logs",
+		Example: "kmesh-daemon logs",
+		Args:    cobra.NoArgs,
+		Run: func(cmd *cobra.Command, args []string) {
+			_ = RunPrintLogs()
```

Comment:
remove, Does it worth doing this?

Reply:
it adds a shortcut for 'cat LOG_FILE_PATH', where LOG_FILE_PATH is the logs file path, like '/var/run/kmesh/daemon.log'. For people not informed of the log path, they can type logs command to print logs.

Reply:
We should remove the print to files in future.